### ISSUE TITLE
Failed to activate the Atom-Hack package

### ISSUE THREAD

**Issue by rujaun** [steelbrain/Atom-Hack#80]
Getting this from installing Atom-Hack from atom's packages

**Atom Version**: 1.8.0
**System**: Ubuntu 16.04
**Thrown From**: [Atom-Hack](https://github.com/steelbrain/atom-hack) package, v6.1.7

### Stack Trace

Failed to activate the Atom-Hack package

```
At ENOENT: no such file or directory, lstat '/home/rujaun/atom:'

Error: ENOENT: no such file or directory, lstat '/home/rujaun/atom:'
    at Error (native)
    at fs.lstatSync (fs.js:887:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:229:16)
    at Object.realpathSync (fs.js:1532:21)
    at Object.fs.realpathSync (ATOM_SHELL_ASAR.js:304:29)
    at /home/rujaun/.atom/packages/Atom-Hack/lib/base.js:33:29
    at Hack.readConfig (/home/rujaun/.atom/packages/Atom-Hack/lib/base.js:72:9)
    at new Hack (/home/rujaun/.atom/packages/Atom-Hack/lib/base.js:24:10)
    at Object.<anonymous> (/home/rujaun/.atom/packages/Atom-Hack/lib/base.js:13:16)
    at Module._compile (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .js] (/usr/share/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.activate (/home/rujaun/.atom/packages/Atom-Hack/lib/main.js:18:17)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:183:20)
    at /usr/share/atom/resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app.asar/src/package.js:92:15)
    at /usr/share/atom/resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/usr/share/atom/resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:538:21)
    at /usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:452:29
    at exit (/usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:73:16)
    at triggerExitCallback (/usr/share/atom/resources/app.asar/src/buffered-process.js:215:47)
    at /usr/share/atom/resources/app.asar/src/buffered-process.js:229:18
    at Socket.<anonymous> (/usr/share/atom/resources/app.asar/src/buffered-process.js:100:18)
    at emitOne (events.js:82:20)
    at Socket.emit (events.js:169:7)
    at Pipe._onclose (net.js:469:12)
```

### Commands

```
     -1:37 tree-view:remove-project-folder (div.header.list-item)
     -1:07.3.0 core:backspace (atom-text-editor#editor.tabLength.editor.mini.is-focused)
     -0:48.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
Atom-Hack, v6.1.7
autocomplete-hack, v2.1.5
language-hack, v0.1.1

# Dev
No dev packages
```

**Comment by steelbrain** [comment]
Hey, any ideas how I can repro this? and was this on a network mount?

**Comment by rujaun** [comment]
Well, I think its my fault as I installed all your hack packages at once
without waiting for one to finish. Hack-Lang I installed last. So I removed
atom and reinstalled it again and waited for each package to install
individually. Works fine now. :)
On 2 Jul 2016 4:11 p.m., "Steel Brain" <notifications@github.com> wrote:

Hey, any ideas how I can repro this? and was this on a network mount?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/steelbrain/Atom-Hack/issues/80#issuecomment-230103605>,
or mute the thread
<https://github.com/notifications/unsubscribe/AA4eW0An4OTJyzFv53vS9pyqgXW7B67mks5qRnGagaJpZM4JDriW>
.
